docs/sources/setup-grafana/configure-security/configure-authentication/ldap-ui/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - ../../../auth/enhanced-ldap/
+description: Learn about configuring LDAP authentication in Grafana using the Grafana UI.
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: LDAP user interface
+title: Configure LDAP authentication using the Grafana user interface
+weight: 300
+---
+
+# Configure LDAP authentication using the Grafana user interface
+
+This page explains how to configure LDAP authentication in Grafana using the Grafana user interface. For more detailed information about configuring LDAP authentication using the configuration file, refer to [LDAP authentication]({{< relref "../ldap" >}}).
+
+Benefits of using the Grafana user interface to configure LDAP authentication include:
+
+- There is no need to edit the configuration file manually.
+- Quickly test the connection to the LDAP server.
+- There is no need to restart Grafana after making changes.
+
+{{% admonition type="note" %}}
+Any configuration changes made through the Grafana user interface (UI) will take precedence over settings specified in the Grafana configuration file or through environment variables. If you modify any configuration settings in the UI, they will override any corresponding settings set via environment variables or defined in the configuration file.
+
+For more information on how Grafana determines the order of precedence for its settings, please refer to the [Settings update at runtime]({{< relref "../../../configure-grafana/settings-updates-at-runtime" >}}).
+{{% /admonition %}}
+
+## Before you begin
+
+Prerequisites:
+
+- Knowledge of LDAP authentication and how it works.
+- Grafana instance v11.3.0 or later.
+- Permissions `settings:read` and `settings:write` with `settings:auth.ldap:*` scope.
+
+## Steps to configure LDAP authentication
+
+Sign in to Grafana and navigate to **Administration > Authentication > LDAP**.
+
+### 1. Complete mandatory fields
+
+The mandatory fields have an asterisk (**\***) next to them. Complete the following fields:
+
+1. **Server host**: Host name or IP address of the LDAP server.
+1. **Search filter**: The LDAP search filter finds entries within the directory.
+1. **Search base DNS**: List of base DNs to search through.
+
+### 2. Complete optional fields
+
+Complete the optional fields as needed:
+
+1. **Bind DN**: Distinguished name (DN) of the user to bind to.
+1. **Bind password**: Password for the server.
+
+### 3. Advanced settings
+
+Click the **Edit** button in the **Advanced settings** section to configure the following settings:
```

Comment:
Since we mention how to open the advanced settings, maybe we should also mention how to go back to the main form from the advanced settings. At the moment this is not very straight forward as users have to click outside of the drawer.

Reply:
I'm afraid I have to disagree with this; it sends a condescending message. The drawer behaviour is standard, with an `X` at the top right corner.